### DISCUSSION TITLE
Related tab doesn’t show the entities that are used in scripts.

### DISCUSSION THREAD

**Discussion by Patrick1610** [home-assistant/frontend#17708] [Dashboard] [answered]
When I open the related tab of a device/entity, the automations that use the entity from an template are not shown.

**Comment by karwosts** [comment, marked as answer]
Frontend actually isn't responsible for calculating this, the home-assistant/core 'search/related' API function returns a list of all the related items for frontend to display.

**Reply by Patrick1610** [reply]
Don I need to make a new issue there?

**Reply by karwosts** [reply]
Probably yes. You can use this integration as the one to file it under: https://www.home-assistant.io/integrations/search/

**Reply by Patrick1610** [reply]
Thanks! I have made another issue
